scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -213,9 +213,9 @@
         </thead>
         <tbody>
           <tr>
-            <td>{{ variant.local_obs_old or 'N/A' }}</td>
-            <td>{{ variant.local_obs_hom_old or 'N/A' }}</td>
-            <td>{{ variant.local_obs_total_old or 'N/A' }}</td>
+            <td>{{ variant.local_obs_old|default('N/A') }}</td>
+            <td>{{ variant.local_obs_hom_old|default('N/A') }}</td>
```

Comment:
And to be frank, oftentimes the `Obs` and `Hom` keys would simply be unset, which this won't fix. We could try to resolve this by adding some convention with MIP crew. Perhaps the new addition of local_obs_total_count from next update will suffice?

Reply:
Perhaps!

Reply:
Nice fix! Reading in the [docs](https://jinja.palletsprojects.com/en/3.0.x/templates/?highlight=default#jinja-filters.default), if local_obs_hom_old exists and it's empty, for instance then nothing will be printed in the page. So what about adding `true`, as a second param to the function, to default to N/A in case the key exists and it's blank?